ci/test-core.sh

```diff
@@ -0,0 +1,72 @@
+# Note(JP): to be `source`d. The reason this is in its own file is that there
+# are (at the time of writing) two consumers. Once for regular testing, and for
+# upgrade-testing. Previously, they were invoking this set of tests with each
+# their own code (duplicated, but not propery duplicated).
+
+# The tenant API authenticator keypair management capability is confirmed to
+# work -- now create a keypair, push the public key into the cluster and
+# generate a tenant API authentication token for a tenant that does not
+# exist yet in the cluster -- use a random name. Then inject into test-remote:
+# - the name of that tenant
+# - the path to the authentication token file
+set -x
+set +e; RNDSTRING=$( tr -dc a-z < /dev/urandom | head -c 6 ); set -e
+TENANT_RND_NAME_FOR_TESTING_ADD_TENANT="testtenant${RNDSTRING}"
+./build/bin/opstrace ta-create-keypair ./ta-custom-keypair.pem
+./build/bin/opstrace ta-create-token "${OPSTRACE_CLUSTER_NAME}" \
+    "${TENANT_RND_NAME_FOR_TESTING_ADD_TENANT}" ta-custom-keypair.pem > tenant-rnd-auth-token-from-custom-keypair
+TENANT_RND_AUTHTOKEN="$(cat tenant-rnd-auth-token-from-custom-keypair)"
+./build/bin/opstrace ta-pubkeys-add \
```

Comment:
The upgrade tests don't build the CLI so this [./build/bin/opstrace binary doesn't exist](https://buildkite.com/opstrace/upgrade-tests/builds/990#4dc318cf-b284-4ec9-aa2f-425a0df75f5f/6597-6602) it should point at `OPSTRACE_CLI_VERSION_TO`.

Reply:
ouch. Yes. Tomorrow. Or, feel free to add to this branch.